frontends/web/src/locales/en/app.json

```diff
@@ -319,7 +319,8 @@
   "bitsuranceAccount": {
     "noAccount": "There are no accounts that can be insured.",
     "select": "Select account",
-    "title": "Insurance"
+    "title": "Insurance",
+    "errorNoXpub": "Error: Was not able to get xpub from account."
```

Comment:
This needs to be fixed (keys should be in alphabetical order) with `make locize-fix` to pass CI tests. You can check on your local with `make ci`

Reply:
Should be fixed now.